xdsl/dialects/ptr.py

```diff
@@ -32,6 +39,9 @@
 class PtrType(ParametrizedAttribute, TypeAttribute):
     name = "ptr_xdsl.ptr"
 
+    def __init__(self):
```

Comment:
Does this do anything?

Reply:
It removes the option of passing arguments to the init function